2024-09-10 17:08:41,787 INFO    StreamThr :532372 [internal.py:wandb_internal():85] W&B internal server running at pid: 532372, started at: 2024-09-10 17:08:41.786793
2024-09-10 17:08:41,790 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: status
2024-09-10 17:08:41,791 INFO    WriterThread:532372 [datastore.py:open_for_write():87] open: /home/ubuntu/frontend/bistro/wandb/run-20240910_170841-8wesq56t/run-8wesq56t.wandb
2024-09-10 17:08:41,792 DEBUG   SenderThread:532372 [sender.py:send():391] send: header
2024-09-10 17:08:41,792 DEBUG   SenderThread:532372 [sender.py:send():391] send: run
2024-09-10 17:08:42,025 INFO    SenderThread:532372 [dir_watcher.py:__init__():211] watching files in: /home/ubuntu/frontend/bistro/wandb/run-20240910_170841-8wesq56t/files
2024-09-10 17:08:42,025 INFO    SenderThread:532372 [sender.py:_start_run_threads():1200] run started: 8wesq56t with start time 1725988121.786299
2024-09-10 17:08:42,032 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: check_version
2024-09-10 17:08:42,032 DEBUG   SenderThread:532372 [sender.py:send_request():418] send_request: check_version
2024-09-10 17:08:42,089 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: run_start
2024-09-10 17:08:42,837 DEBUG   HandlerThread:532372 [system_info.py:__init__():26] System info init
2024-09-10 17:08:42,837 DEBUG   HandlerThread:532372 [system_info.py:__init__():41] System info init done
2024-09-10 17:08:42,837 INFO    HandlerThread:532372 [system_monitor.py:start():194] Starting system monitor
2024-09-10 17:08:42,838 INFO    SystemMonitor:532372 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-09-10 17:08:42,838 INFO    HandlerThread:532372 [system_monitor.py:probe():214] Collecting system info
2024-09-10 17:08:42,839 INFO    SystemMonitor:532372 [interfaces.py:start():188] Started cpu monitoring
2024-09-10 17:08:42,840 INFO    SystemMonitor:532372 [interfaces.py:start():188] Started disk monitoring
2024-09-10 17:08:42,841 INFO    SystemMonitor:532372 [interfaces.py:start():188] Started gpu monitoring
2024-09-10 17:08:42,842 INFO    SystemMonitor:532372 [interfaces.py:start():188] Started memory monitoring
2024-09-10 17:08:42,843 INFO    SystemMonitor:532372 [interfaces.py:start():188] Started network monitoring
2024-09-10 17:08:43,481 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:43,482 DEBUG   HandlerThread:532372 [system_info.py:probe():152] Probing system
2024-09-10 17:08:43,484 DEBUG   HandlerThread:532372 [system_info.py:_probe_git():137] Probing git
2024-09-10 17:08:43,489 DEBUG   HandlerThread:532372 [system_info.py:_probe_git():145] Probing git done
2024-09-10 17:08:43,489 DEBUG   HandlerThread:532372 [system_info.py:probe():200] Probing system done
2024-09-10 17:08:43,489 DEBUG   HandlerThread:532372 [system_monitor.py:probe():223] {'os': 'Linux-5.19.0-1010-nvidia-lowlatency-x86_64-with-glibc2.35', 'python': '3.10.12', 'heartbeatAt': '2024-09-10T17:08:43.482460', 'startedAt': '2024-09-10T17:08:41.782775', 'docker': None, 'cuda': None, 'args': ('--wallet.name', 'Alice', '--wallet.hotkey', 'Charlie', '--subtensor.network', 'test', '--device', 'cuda:3', '--use_wandb'), 'state': 'running', 'program': '/home/ubuntu/frontend/bistro/miner.py', 'codePathLocal': 'miner.py', 'codePath': 'bistro/miner.py', 'git': {'remote': 'https://github.com/unconst/frontend.git', 'commit': 'a0660981f23476af1751d70673e4586879b6970c'}, 'email': 'jake@bittensor.com', 'root': '/home/ubuntu/frontend', 'host': 'training', 'username': 'ubuntu', 'executable': '/home/ubuntu/frontend/venv/bin/python3', 'cpu_count': 64, 'cpu_count_logical': 128, 'cpu_freq': {'current': 2380.601546875, 'min': 1500.0, 'max': 3250.0}, 'cpu_freq_per_core': [{'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1515.188, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3800.127, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3791.064, 'min': 1500.0, 'max': 3250.0}, {'current': 3799.381, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 2300.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3798.634, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3798.168, 'min': 1500.0, 'max': 3250.0}, {'current': 3787.966, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3790.134, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3800.268, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3799.002, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3805.157, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 2300.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3800.138, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3780.133, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3800.119, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3789.945, 'min': 1500.0, 'max': 3250.0}, {'current': 3803.048, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3800.117, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 2171.509, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3250.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 1500.0, 'min': 1500.0, 'max': 3250.0}, {'current': 3800.035, 'min': 1500.0, 'max': 3250.0}], 'disk': {'/': {'total': 3519.6199798583984, 'used': 90.96326446533203}}, 'gpu': 'NVIDIA H100 PCIe', 'gpu_count': 8, 'memory': {'total': 1511.5785369873047}}
2024-09-10 17:08:43,489 INFO    HandlerThread:532372 [system_monitor.py:probe():224] Finished collecting system info
2024-09-10 17:08:43,489 INFO    HandlerThread:532372 [system_monitor.py:probe():227] Publishing system info
2024-09-10 17:08:43,490 INFO    HandlerThread:532372 [system_monitor.py:probe():229] Finished publishing system info
2024-09-10 17:08:43,492 DEBUG   SenderThread:532372 [sender.py:send():391] send: files
2024-09-10 17:08:43,492 INFO    SenderThread:532372 [sender.py:_save_file():1466] saving file wandb-metadata.json with policy now
2024-09-10 17:08:43,512 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:43,594 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: python_packages
2024-09-10 17:08:43,594 DEBUG   SenderThread:532372 [sender.py:send_request():418] send_request: python_packages
2024-09-10 17:08:43,595 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: stop_status
2024-09-10 17:08:43,595 DEBUG   SenderThread:532372 [sender.py:send_request():418] send_request: stop_status
2024-09-10 17:08:43,597 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:08:43,733 DEBUG   SenderThread:532372 [sender.py:send():391] send: telemetry
2024-09-10 17:08:43,779 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:43,779 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:43,779 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:43,779 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:43,837 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:44,028 INFO    Thread-12 :532372 [dir_watcher.py:_on_file_created():271] file/dir created: /home/ubuntu/frontend/bistro/wandb/run-20240910_170841-8wesq56t/files/wandb-metadata.json
2024-09-10 17:08:44,028 INFO    Thread-12 :532372 [dir_watcher.py:_on_file_created():271] file/dir created: /home/ubuntu/frontend/bistro/wandb/run-20240910_170841-8wesq56t/files/requirements.txt
2024-09-10 17:08:44,435 INFO    wandb-upload_0:532372 [upload_job.py:push():130] Uploaded file /tmp/tmpctnbssdbwandb/fvl0ugtn-wandb-metadata.json
2024-09-10 17:08:44,663 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:08:45,642 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:08:45,837 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:45,837 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:45,837 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:45,837 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:45,837 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:45,837 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:45,837 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:45,936 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:08:45,937 DEBUG   SenderThread:532372 [sender.py:send():391] send: history
2024-09-10 17:08:45,938 DEBUG   SenderThread:532372 [sender.py:send_request():418] send_request: summary_record
2024-09-10 17:08:45,940 INFO    SenderThread:532372 [sender.py:_save_file():1466] saving file wandb-summary.json with policy end
2024-09-10 17:08:46,027 INFO    Thread-12 :532372 [dir_watcher.py:_on_file_created():271] file/dir created: /home/ubuntu/frontend/bistro/wandb/run-20240910_170841-8wesq56t/files/wandb-summary.json
2024-09-10 17:08:46,027 INFO    Thread-12 :532372 [dir_watcher.py:_on_file_created():271] file/dir created: /home/ubuntu/frontend/bistro/wandb/run-20240910_170841-8wesq56t/files/output.log
2024-09-10 17:08:46,643 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:08:47,002 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: status_report
2024-09-10 17:08:47,643 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:08:47,838 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:47,838 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:47,838 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:47,838 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:47,838 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:47,838 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:47,838 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:48,012 ERROR   FileStreamThread:532372 [file_stream.py:_handle_response():510] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/const69/bistro/8wesq56t/file_stream
NoneType: None
2024-09-10 17:08:48,028 INFO    Thread-12 :532372 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/ubuntu/frontend/bistro/wandb/run-20240910_170841-8wesq56t/files/output.log
2024-09-10 17:08:48,643 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:08:49,643 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:08:50,190 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:50,192 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:50,192 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:50,192 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:50,192 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:50,192 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:50,192 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:50,643 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:08:52,229 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:52,261 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:52,261 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:52,261 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:52,262 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:52,262 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:52,262 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:52,597 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: status_report
2024-09-10 17:08:54,320 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:54,409 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:54,409 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:54,409 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:54,409 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:54,409 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:54,409 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:56,441 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:56,446 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:56,447 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:56,447 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:56,447 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:56,447 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:56,447 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:57,598 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: status_report
2024-09-10 17:08:59,301 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:59,303 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:59,304 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:59,304 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:59,304 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:59,304 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:08:59,304 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:01,307 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:01,324 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:01,324 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:01,324 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:01,324 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:01,324 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:01,324 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:02,598 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: status_report
2024-09-10 17:09:04,393 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:04,452 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:04,453 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:04,453 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:04,453 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:04,453 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:04,453 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:04,679 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: stop_status
2024-09-10 17:09:04,680 DEBUG   SenderThread:532372 [sender.py:send_request():418] send_request: stop_status
2024-09-10 17:09:04,720 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:05,679 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:06,515 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:06,555 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:06,555 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:06,555 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:06,555 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:06,555 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:06,555 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:06,679 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:07,680 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:07,774 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: status_report
2024-09-10 17:09:08,572 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:08,579 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:08,579 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:08,579 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:08,579 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:08,579 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:08,579 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:08,679 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:09,679 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:10,511 ERROR   FileStreamThread:532372 [file_stream.py:_handle_response():510] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/const69/bistro/8wesq56t/file_stream
NoneType: None
2024-09-10 17:09:10,646 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:10,675 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:10,675 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:10,675 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:10,675 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:10,675 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:10,675 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:10,680 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:11,036 INFO    Thread-12 :532372 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/ubuntu/frontend/bistro/wandb/run-20240910_170841-8wesq56t/files/output.log
2024-09-10 17:09:11,680 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:12,680 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:12,716 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:12,718 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:12,718 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:12,718 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:12,718 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:12,718 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:12,718 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:13,685 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: status_report
2024-09-10 17:09:13,686 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:13,718 ERROR   SenderThread:532372 [internal_api.py:execute():349] 409 response executing GraphQL.
2024-09-10 17:09:13,718 ERROR   SenderThread:532372 [internal_api.py:execute():350] {"errors":[{"message":"run 8wesq56t was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-09-10 17:09:14,720 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:14,819 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:14,869 ERROR   SenderThread:532372 [internal_api.py:execute():349] 409 response executing GraphQL.
2024-09-10 17:09:14,869 ERROR   SenderThread:532372 [internal_api.py:execute():350] {"errors":[{"message":"run 8wesq56t was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-09-10 17:09:14,945 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:14,945 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:14,945 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:14,945 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:14,945 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:14,945 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:15,681 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:16,681 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:16,946 ERROR   gpu       :532372 [interfaces.py:aggregate():159] Failed to serialize metric: division by zero
2024-09-10 17:09:16,989 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:17,045 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:17,045 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:17,045 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:17,045 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:17,045 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:17,045 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:17,069 ERROR   SenderThread:532372 [internal_api.py:execute():349] 409 response executing GraphQL.
2024-09-10 17:09:17,069 ERROR   SenderThread:532372 [internal_api.py:execute():350] {"errors":[{"message":"run 8wesq56t was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-09-10 17:09:17,069 INFO    SenderThread:532372 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 345, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/requests/models.py", line 1024, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 409 Client Error: Conflict for url: https://api.wandb.ai/graphql
2024-09-10 17:09:17,681 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:18,681 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:19,222 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:19,357 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:19,357 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:19,357 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:19,357 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:19,357 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:19,357 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:19,678 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: stop_status
2024-09-10 17:09:19,718 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:20,682 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:21,397 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:21,444 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:21,445 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:21,445 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:21,445 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:21,445 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:21,445 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:21,623 ERROR   SenderThread:532372 [internal_api.py:execute():349] 409 response executing GraphQL.
2024-09-10 17:09:21,624 ERROR   SenderThread:532372 [internal_api.py:execute():350] {"errors":[{"message":"run 8wesq56t was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-09-10 17:09:21,682 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:22,682 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:23,516 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:23,640 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:23,641 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:23,641 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:23,641 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:23,641 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:23,641 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:23,682 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:24,682 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:25,682 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:25,684 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:25,764 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:25,765 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:25,765 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:25,765 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:25,765 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:25,765 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:26,683 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:27,683 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:27,866 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:27,968 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:27,968 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:27,968 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:27,968 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:27,968 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:27,968 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:28,683 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:30,018 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:30,021 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:30,021 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:30,021 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:30,021 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:30,021 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:30,021 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:30,053 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:31,011 ERROR   SenderThread:532372 [internal_api.py:execute():349] 409 response executing GraphQL.
2024-09-10 17:09:31,011 ERROR   SenderThread:532372 [internal_api.py:execute():350] {"errors":[{"message":"run 8wesq56t was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-09-10 17:09:31,053 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:32,053 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:32,119 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:32,193 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:32,194 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:32,194 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:32,194 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:32,194 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:32,194 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:33,053 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:34,053 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:34,210 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:34,239 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:34,239 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:34,239 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:34,239 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:34,239 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:34,239 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:35,054 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:36,384 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:36,480 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:36,480 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:36,480 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:36,480 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:36,481 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:36,481 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:36,609 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:37,608 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:38,515 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:38,557 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:38,557 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:38,557 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:38,557 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:38,557 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:38,557 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:38,609 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:39,609 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:40,609 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:40,650 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:40,683 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:40,683 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:40,683 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:40,683 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:40,683 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:40,683 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:41,609 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:42,609 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:42,778 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:42,843 DEBUG   SystemMonitor:532372 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-09-10 17:09:42,868 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:42,869 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:42,869 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:42,869 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:42,869 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:42,869 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:43,610 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:44,871 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:44,908 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:44,910 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:44,910 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:44,910 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:44,910 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:44,910 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:44,910 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:45,871 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:46,873 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:46,964 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:47,032 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:47,032 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:47,032 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:47,032 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:47,032 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:47,033 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:47,872 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:48,872 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:49,033 ERROR   gpu       :532372 [interfaces.py:aggregate():159] Failed to serialize metric: division by zero
2024-09-10 17:09:49,121 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:49,186 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:49,186 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:49,186 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:49,186 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:49,186 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:49,186 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:49,635 ERROR   SenderThread:532372 [internal_api.py:execute():349] 409 response executing GraphQL.
2024-09-10 17:09:49,636 ERROR   SenderThread:532372 [internal_api.py:execute():350] {"errors":[{"message":"run 8wesq56t was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-09-10 17:09:49,872 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:50,873 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:51,276 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:51,390 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:51,390 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:51,390 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:51,390 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:51,390 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:51,390 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:51,873 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:52,873 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:53,463 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:53,557 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:53,557 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:53,557 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:53,557 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:53,557 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:53,557 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:53,873 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:54,873 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:55,641 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:55,742 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:55,742 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:55,742 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:55,742 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:55,742 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:55,742 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:55,873 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:56,873 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:57,795 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:57,874 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:57,893 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:57,893 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:57,893 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:57,893 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:57,894 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:57,894 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:09:58,874 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:09:59,874 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:00,010 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:00,094 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:00,094 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:00,094 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:00,094 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:00,094 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:00,094 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:00,874 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:01,874 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:02,128 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:02,157 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:02,157 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:02,158 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:02,158 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:02,158 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:02,158 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:02,875 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:03,875 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:04,310 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:04,498 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:04,498 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:04,498 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:04,498 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:04,498 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:04,498 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:04,875 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:05,875 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:06,543 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:06,648 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:06,649 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:06,649 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:06,649 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:06,649 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:06,649 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:06,875 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:07,876 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:08,743 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:08,835 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:08,835 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:08,835 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:08,835 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:08,835 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:08,835 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:08,876 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:09,876 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:10,876 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:10,897 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:10,950 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:10,951 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:10,951 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:10,951 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:10,951 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:10,951 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:11,877 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:12,876 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:12,959 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:12,961 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:12,961 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:12,961 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:12,961 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:12,961 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:12,961 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:13,877 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:14,877 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:14,974 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:15,069 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:15,070 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:15,070 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:15,070 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:15,070 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:15,070 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:15,877 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:16,877 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:17,101 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:17,135 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:17,135 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:17,135 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:17,135 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:17,135 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:17,135 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:17,877 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:18,878 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:19,188 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:19,242 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:19,242 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:19,242 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:19,242 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:19,242 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:19,242 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:19,878 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:20,878 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:21,243 ERROR   gpu       :532372 [interfaces.py:aggregate():159] Failed to serialize metric: division by zero
2024-09-10 17:10:21,274 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:21,337 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:21,337 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:21,337 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:21,337 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:21,337 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:21,337 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:21,879 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:22,879 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:23,339 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:23,363 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:23,363 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:23,364 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:23,364 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:23,364 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:23,364 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:23,879 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:24,879 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:25,409 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:25,455 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:25,455 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:25,455 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:25,455 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:25,455 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:25,456 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:25,879 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:26,880 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:27,489 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:27,547 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:27,547 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:27,547 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:27,548 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:27,548 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:27,548 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:27,880 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:28,880 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:29,043 ERROR   SenderThread:532372 [internal_api.py:execute():349] 409 response executing GraphQL.
2024-09-10 17:10:29,043 ERROR   SenderThread:532372 [internal_api.py:execute():350] {"errors":[{"message":"run 8wesq56t was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-09-10 17:10:29,550 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:29,550 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:29,550 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:29,550 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:29,550 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:29,550 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:29,550 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:31,561 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:31,563 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:31,563 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:31,563 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:31,563 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:31,563 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:31,564 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:33,567 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:33,572 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:33,573 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:33,573 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:33,573 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:33,573 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:33,573 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:33,951 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:34,853 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:35,573 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:35,573 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:35,574 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:35,574 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:35,574 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:35,574 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:35,574 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:35,853 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:36,854 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:37,422 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:37,574 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:37,574 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:37,574 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:37,575 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:37,575 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:37,575 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:37,575 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:37,854 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:38,854 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:39,130 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:39,433 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:39,575 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:39,575 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:39,575 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:39,575 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:39,575 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:39,576 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:39,576 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:39,717 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:39,854 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:40,008 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:40,297 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:40,586 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:40,854 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:40,875 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:41,164 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:41,453 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:41,576 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:41,576 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:41,576 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:41,576 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:41,576 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:41,576 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:41,576 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:41,742 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:41,854 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:42,031 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:42,320 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:42,610 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:42,854 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:42,898 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:43,187 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:43,476 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:43,577 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:43,577 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:43,577 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:43,577 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:43,577 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:43,577 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:43,577 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:43,765 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:43,854 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:44,054 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:44,343 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:44,633 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:44,855 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:44,921 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:45,211 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:45,500 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:45,577 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:45,578 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:45,578 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:45,578 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:45,578 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:45,578 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:45,578 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:45,789 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:45,855 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:46,078 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:46,855 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:47,578 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:47,578 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:47,578 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:47,579 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:47,579 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:47,579 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:47,579 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:47,856 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:48,856 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:49,579 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:49,579 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:49,579 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:49,579 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:49,579 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:49,580 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:49,580 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:49,856 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:50,856 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:51,580 ERROR   gpu       :532372 [interfaces.py:aggregate():159] Failed to serialize metric: division by zero
2024-09-10 17:10:51,580 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:51,580 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:51,580 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:51,580 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:51,580 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:51,581 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:51,581 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:51,857 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:52,857 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:53,581 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:53,581 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:53,581 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:53,581 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:53,581 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:53,581 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:53,582 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:53,857 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:54,725 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:54,857 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:55,010 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:55,295 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:55,581 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:55,582 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:55,582 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:55,582 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:55,582 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:55,582 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:55,582 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:55,582 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:55,857 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:55,870 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:56,159 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:56,448 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:56,738 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:56,857 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:57,028 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:57,317 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:57,583 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:57,583 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:57,583 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:57,583 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:57,583 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:57,583 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:57,583 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:57,607 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:57,858 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:57,896 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:58,185 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:58,475 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:58,765 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:58,858 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:59,054 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:59,344 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:59,583 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:59,584 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:59,584 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:59,584 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:59,584 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:59,584 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:59,584 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:10:59,633 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:10:59,858 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:10:59,922 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:00,212 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:00,502 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:00,791 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:00,858 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:01,081 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:01,370 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:01,584 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:01,584 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:01,584 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:01,584 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:01,584 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:01,584 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:01,584 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:01,659 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:01,858 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:02,858 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:03,585 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:03,585 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:03,585 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:03,585 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:03,585 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:03,585 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:03,585 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:03,858 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:04,859 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:05,585 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:05,586 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:05,586 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:05,586 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:05,586 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:05,586 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:05,586 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:05,859 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:06,859 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:07,586 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:07,586 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:07,587 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:07,587 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:07,587 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:07,587 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:07,587 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:07,859 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:08,859 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:09,207 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:09,496 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:09,587 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:09,587 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:09,587 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:09,587 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:09,588 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:09,588 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:09,588 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:09,787 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:09,859 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:10,077 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:10,366 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:10,656 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:10,859 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:10,945 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:11,234 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:11,526 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:11,588 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:11,588 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:11,588 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:11,588 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:11,588 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:11,588 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:11,588 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:11,815 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:11,859 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:12,104 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:12,393 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:12,682 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:12,859 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:12,972 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:13,261 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:13,551 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:13,589 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:13,589 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:13,589 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:13,589 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:13,589 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:13,589 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:13,589 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:13,840 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:13,860 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:14,130 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:14,419 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:14,708 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:14,860 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:14,997 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:15,287 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:15,576 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:15,590 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:15,590 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:15,590 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:15,590 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:15,590 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:15,590 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:15,590 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:15,860 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:15,865 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:16,154 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: partial_history
2024-09-10 17:11:16,860 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:17,591 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:17,591 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:17,591 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:17,591 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:17,591 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:17,591 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:17,591 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:17,860 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:18,860 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:19,591 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:19,592 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:19,592 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:19,592 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:19,592 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:19,592 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:19,592 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:19,860 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:20,861 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:21,592 ERROR   gpu       :532372 [interfaces.py:aggregate():159] Failed to serialize metric: division by zero
2024-09-10 17:11:21,592 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:21,593 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:21,593 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:21,593 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:21,593 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:21,593 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:21,593 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:21,871 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:22,866 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:23,593 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:23,593 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:23,593 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:23,593 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:23,593 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:23,593 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:23,593 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:23,866 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:24,866 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:25,594 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:25,594 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:25,594 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:25,594 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:25,594 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:25,594 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:25,594 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:25,867 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:26,867 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:27,594 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:27,595 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:27,595 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:27,595 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:27,595 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:27,595 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:27,595 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:27,867 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:28,867 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:29,595 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:29,595 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:29,595 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:29,595 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:29,596 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:29,596 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:29,596 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:29,867 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:30,867 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:31,596 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:31,596 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:31,596 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:31,596 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:31,596 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:31,596 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:31,596 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:31,867 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:32,867 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:33,597 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:33,597 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:33,597 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:33,597 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:33,597 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:33,597 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:33,597 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:33,867 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:34,867 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:35,598 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:35,598 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:35,599 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:35,599 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:35,599 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:35,599 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:35,599 ERROR   gpu       :532372 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-10 17:11:35,868 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:35,993 ERROR   SenderThread:532372 [internal_api.py:execute():349] 409 response executing GraphQL.
2024-09-10 17:11:35,993 ERROR   SenderThread:532372 [internal_api.py:execute():350] {"errors":[{"message":"run 8wesq56t was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-09-10 17:11:36,845 INFO    WriterThread:532372 [datastore.py:close():296] close: /home/ubuntu/frontend/bistro/wandb/run-20240910_170841-8wesq56t/run-8wesq56t.wandb
2024-09-10 17:11:36,868 DEBUG   HandlerThread:532372 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-10 17:11:36,868 INFO    HandlerThread:532372 [handler.py:finish():880] shutting down handler
2024-09-10 17:11:36,868 INFO    HandlerThread:532372 [system_monitor.py:finish():203] Stopping system monitor
2024-09-10 17:11:36,868 ERROR   gpu       :532372 [interfaces.py:aggregate():159] Failed to serialize metric: division by zero
2024-09-10 17:11:36,870 DEBUG   SystemMonitor:532372 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-09-10 17:11:36,870 INFO    HandlerThread:532372 [interfaces.py:finish():200] Joined cpu monitor
2024-09-10 17:11:36,870 DEBUG   SystemMonitor:532372 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-09-10 17:11:36,870 INFO    HandlerThread:532372 [interfaces.py:finish():200] Joined disk monitor
2024-09-10 17:11:36,873 INFO    HandlerThread:532372 [interfaces.py:finish():200] Joined gpu monitor
2024-09-10 17:11:36,873 INFO    HandlerThread:532372 [interfaces.py:finish():200] Joined memory monitor
2024-09-10 17:11:36,873 INFO    HandlerThread:532372 [interfaces.py:finish():200] Joined network monitor
2024-09-10 17:11:36,971 ERROR   StreamThr :532372 [internal.py:wandb_internal():173] Thread SenderThread:
Traceback (most recent call last):
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2224, in upsert_run
    response = self.gql(
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 317, in gql
    ret = self._retry_gql(
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 345, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/requests/models.py", line 1024, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 409 Client Error: Conflict for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/sdk/internal/internal_util.py", line 48, in run
    self._run()
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/sdk/internal/internal_util.py", line 92, in _run
    self._debounce()
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/sdk/internal/internal.py", line 333, in _debounce
    self._sm.debounce()
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 549, in debounce
    self._maybe_update_config(always=final)
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 526, in _maybe_update_config
    self._debounce_config()
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 555, in _debounce_config
    self._api.upsert_run(
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/wandb/apis/normalize.py", line 51, in wrapper
    raise CommError(message, error)
wandb.errors.CommError: run 8wesq56t was previously created and deleted; try a new run name (Error 409: Conflict)
